src/uu/tail/src/tail.rs

```diff
@@ -9,23 +9,35 @@
 
 // spell-checker:ignore (ToDO) seekable seek'd tail'ing ringbuffer ringbuf
 
+#[macro_use]
+extern crate clap;
+
 #[macro_use]
 extern crate uucore;
 
 mod platform;
 
+use clap::{App, Arg};
 use std::collections::VecDeque;
 use std::error::Error;
 use std::fmt;
 use std::fs::File;
 use std::io::{stdin, stdout, BufRead, BufReader, Read, Seek, SeekFrom, Write};
 use std::path::Path;
-use std::str::from_utf8;
 use std::thread::sleep;
 use std::time::Duration;
 
-static NAME: &str = "tail";
-static VERSION: &str = env!("CARGO_PKG_VERSION");
+static OPT_BYTES: &str = "bytes";
```

Comment:
Please declare them in alphabetic order :)
The other maintainer prefers it this way